js/scenes/Home.js

```diff
@@ -436,6 +442,12 @@ export class Home extends Component {
     );
   }
 
+  renderHeader() {
+    return (
+      <View style={styles.firstRow}/>
```

Comment:
This element overhangs the header content and blocks the header buttons. 'Settings' button is difficult to press.